### PR TITLE
refs #16008 - Add support for configurable puppet_content_dir

### PR DESCRIPTION
**Pull request by stbenjam**

### CODE REVIEW

**Comment by stbenjam** [comment]
Needs https://github.com/theforeman/smart-proxy-pulp/pull/8 to be merged first

**Comment by stbenjam** [comment]
Green: http://ci.theforeman.org/job/test_proxy_plugin_pull_request/132/

@witlessbird or @ohadlevy Would greatly appreciate a review on this, we'd like to get puppet 4 wrapped up in time for branching next week.

Thanks!

**Comment by shlomizadok** [comment]
LGTM  :+1:

File: Gemfile [hunk 1]
```diff
@@ -4,3 +4,6 @@ gemspec
 group :development do
   gem 'smart_proxy', :git => 'https://github.com/theforeman/smart-proxy', :branch => 'develop'
 end
+
+gem 'addressable', '< 2.4.0',  {"platforms"=>["ruby_18"]}
+gem 'json', '~> 1.0', {"platforms"=>["ruby_18", "ruby_19"]}
```
Comment (witlessbird) [inline Gemfile]: No need for curvy braces here and in the line above.

**Comment by stbenjam** [comment]
@witlessbird Fixed, green again: http://ci.theforeman.org/job/test_proxy_plugin_pull_request/135/

**Comment by witlessbird** [comment]
Merged as 99a37ef. Thanks, @stbenjam.

**Pull request closed without merging**